ui/lib/config-ui/addon/components/messages/page/details.hbs

```diff
@@ -43,6 +43,10 @@
         (date-format (get @message attr.name) "MMM d, yyyy hh:mm aaa" withTimeZone=true)
       }}
     />
+  {{else if (eq attr.name "link")}}
+    {{#each-in @message.link as |title|}}
+      <InfoTableRow @label="Link" @value={{if title true false}} />
+    {{/each-in}}
```

Comment:
Does this render `true` if a link exists but doesn't actually display the link content? It seems like rendering the title and href would be useful 🤔

Reply:
<img width="1152" alt="Screenshot 2024-01-18 at 10 06 16 AM" src="https://github.com/hashicorp/vault/assets/30884335/2124b645-2661-41d1-ad5a-d1abf29f859f">
It'll render "Yes" or "No" based on whether or not a link exists. I'll follow up with design on how it should look!

Reply:
Edit: spoke with design and we decided to add the link!
![Screenshot 2024-01-18 at 11 22 21 AM](https://github.com/hashicorp/vault/assets/30884335/5213bdb5-a002-41a7-adc2-4f3d9b817fc6)